test/test-lookup.js

```diff
@@ -49,7 +49,7 @@ test('lookup[getLookupTable]:', function (t) {
   }
   t.ok(table, 'table should exist');
   t.ok(table.lodash, 'lodash should be in the table');
-  t.ok(table.lodash.replace, 'lodash should need to be replaced');
+  t.ok(table.underscore.master, 'underscore should contain a master paramter');
```

Comment:
we cannot go adding replace to our lookup.json, The idea is that we are moving away from using replace. Also all this test does is checks if the tag exists, NOT whether it works